activiti-spring-project/src/main/java/org/activiti/core/common/spring/project/conf/ProjectModelAutoConfiguration.java

```diff
@@ -20,12 +20,10 @@ public ObjectMapper objectMapper() {
     }
 
     @Bean
-    public ProjectModelService projectModelService(@Value("${application.manifest.file.path:classpath:/}") String path,
-                                                   @Value("${activiti.cloud.application.name:default-app}") String applicationName,
+    public ProjectModelService projectModelService(@Value("${application.manifest.file.path:classpath:/default-app.json}") String absolutePath,
```

Comment:
i don't think we need the default value `:classpath:/default-app.json` above, as it is in the property file.